packages/venia-ui/lib/components/WishlistPage/wishlistPage.js

```diff
@@ -27,7 +27,10 @@ const WishlistPage = props => {
         defaultMessage: 'The wishlist is not currently available.'
     });
     const wishlistElements = useMemo(() => {
-        if (wishlists.length === 0) return null;
+        if (wishlists.length === 0) {
+            return <Wishlist data={[]} />;
```

Comment:
With all the destructuring happening in the component, was surprised to see it not complain about this array value. Instead of making the placeholder dependent on the developer passing some arbitrary special value, let's instead make `<Wishlist />` with no data prop behave like a placeholder. That paired with adding `defaultProps` to the component should accomplish the same thing with a better API.

Optionally, it looks like I missed adding `propTypes` on that component. Would be nice to document if you are going to pass data, what is expected in that object to render correctly.